src/services/upload_terminal.js

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright: Ambrosus Technologies GmbH
+Email: tech@ambrosus.com
+
+This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+This Source Code Form is “Incompatible With Secondary Licenses”, as defined by the Mozilla Public License, v. 2.0.
+*/
+
+import {getDefaultAddress} from '../utils/web3_tools';
+
+
+export default class UploadTerminal {
```

Comment:
Personally, I would prefer `UplaodRepository` as this is a access point to a storage solutions, just as the EntityRepositories are.